.github/workflows/autodocs.yml

```diff
@@ -0,0 +1,33 @@
+name: Automatic Documentation for Launcher
+
+on:
+  push:
+    branches:
+      - launcher_docs
+
+jobs:
+  update_docs:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    
+    - name: Install Launcher
+      id: install-launcher
+      run: cargo install --git https://github.com/huggingface/text-generation-inference.git text-generation-launcher
```

Comment:
I don't think GA has cargo by default. Would you be able to trigger the workflow in this PR so we can see the results? (right now `launched.md` seems empty)

Reply:
`text-generation-launcher` command would've failed then, I can see the output in the GHActions logs (I did echo for sanity check)

Reply:
`diff: launcher.md: No such file or directory` this is what fails, it's because in main branch currently it doesn't exist.

Reply:
I'll add launcher.md for now since in GHactions it says `diff: launcher.md: No such file or directory` so it doesn't fall into the if

Reply:
it's the file paths that cause it to not fall into if statement (diff says there's no file) so I fixed it

Reply:
it writes the file I can echo file content and verify but doesn't commit to this branch for some reason

Reply:
Don't  you need to install the current PR branch? (This install main I think)

Reply:
I think last time we agreed on checking it after the PR would be merged to main, let me change this.